src/main/java/com/ss/ita/google/ui/pages/SearchResultPage.java

```diff
@@ -29,10 +28,8 @@ public String getLinkText(int numberOfLink) {
         return getSearchedLink(numberOfLink).getText();
     }
 
-    public SearchResultPage changeFirstLinkColor(String color){
-        JavascriptExecutor js = (JavascriptExecutor) driver;
-        js.executeScript("document.querySelector('.g>div>div>div>a>h3').style.color =" + color +";");
-        return this;
+    public void changeLinkColor(String color, int numberOfLink){
+        executeScriptForWebElementsColor(color,getSearchedLink(numberOfLink));
```

Comment:
this is still to specific, only works for results links, pass locator here instead of link number